Cargo.toml

```diff
@@ -69,7 +69,11 @@ members = [
     "src/utils/workspace-config",
     "src/workspace-hack",
 ]
-exclude = ["e2e_test/source_inline/pubsub", "e2e_test/udf/wasm", "lints"]
```

Comment:
In the future we can add a formatter config file in the project root in the future